src/openrct2/scripting/bindings/game/ScContext.hpp

```diff
@@ -425,6 +425,12 @@ namespace OpenRCT2::Scripting
             return GetIconByName(iconName);
         }
 
+        void setGameSpeed(int32_t speed)
+        {
+            gGameSpeed = std::max(1, (speed > 4 ? (gConfigGeneral.DebuggingTools ? 8 : 4) : speed));
```

Comment:
This would be dangerous to run from a networked game. I think thats the main reason why we haven't implemented it. @ZehMatt any thoughts on this?

Reply:
In theory its safe as clients can't progress past the server tick, however the game speed is not properly synchronized then clients will only fast-forward if they are too far behind the server tick. Ideally this is a game action so clients will progress at the same pace as the server.